src/services/user.service.ts

```diff
@@ -39,4 +40,8 @@ export class UserService {
       order: ['login'],
     });
   }
+
+  async validateUser({ login, password }: LoginUserModel) {
+    return await User.findOne({ where: { login, password } });
```

Comment:
Passwords are not saved in DB with clear text, they are usually hashed

Reply:
Thanks. I know that usually it crypted by bcrypt but I wanted to keep it simple and no requirements were provided regarding password hashing )